src/module-elasticsuite-catalog/Model/Product/Indexer/Fulltext/Datasource/AttributeData.php

```diff
@@ -226,8 +226,11 @@ private function addIndexedAttribute(&$productIndexData, $attribute)
             $productIndexData['indexed_attributes'] = [];
         }
 
-        // Data can be missing for this attribute (Eg : due to null value being escaped).
-        if (isset($productIndexData[$attribute->getAttributeCode()])) {
+        // Data can be missing for this attribute (Eg : due to null value being escaped,
+        // or this attribute is already included in the array)
```

Comment:
Hello @francescosalvi, could you add a dot (".") at the end of that comment line ?